packages/react-query/src/types.ts

```diff
@@ -43,6 +43,16 @@ export interface UseQueryOptions<
     TQueryKey
   > {}
 
+export interface UseSuspenseQueryOptions<
+  TQueryFnData = unknown,
+  TError = unknown,
+  TData = TQueryFnData,
+  TQueryKey extends QueryKey = QueryKey,
+> extends Omit<
+    UseQueryOptions<TQueryFnData, TError, TData, TQueryKey>,
+    'enabled' | 'suspense' | 'placeholderData'
```

Comment:
`placeholderData` and `keepPreviousData` were merged in v5

```suggestion
    'enabled' | 'suspense' | 'placeholderData' | 'keepPreviousData'
```